language/tools/move-cli/src/lib.rs

```diff
@@ -210,11 +210,27 @@ pub enum ExperimentalCommand {
         args: Vec<TransactionArgument>,
         #[structopt(long = "type-args", parse(try_from_str = parser::parse_type_tag))]
         type_args: Vec<TypeTag>,
-        #[structopt(long = "concretize")]
-        concretize: bool,
+        #[structopt(long = "concretize", possible_values = &ConcretizeMode::variants(), case_insensitive = true, default_value = "dont")]
+        concretize: ConcretizeMode,
     },
 }
 
+// Specify if/how the analysis should concretize and filter the static analysis summary
+arg_enum! {
+    // Specify if/how the analysis should concretize and filter the static analysis summary
+    #[derive(Debug, Clone, Copy)]
+    pub enum ConcretizeMode {
+        // Show the full concretized access paths read or written (e.g. 0xA/0x1::M::S/f/g)
```

Comment:
Will those comments show up in cli helper?

Reply:
I think they will not, but Rust won't let me put doc comments inside a macro... Not sure what the best fix is.